_sass/vrsite/fool.scss

```diff
@@ -0,0 +1,67 @@
+.fool-modal-wrapper {
+	display: none;
+
+	position: fixed;
+	z-index: 1;
+	background: rgba($dark-background-color, .7);
+	width: 100vw;
+	height: 100vh;
+	.fool-modal {
+		margin: 50vh auto 0;
+		transform: translateY(-50%);
+		width: fit-content;
+		height: fit-content;
+		background: black;
+		padding: 48px;
+		.closebtn, .closebtn:hover {
+			display: none;
+			font-size: 1.5em;
+			text-decoration: none;
+			position: absolute;
+			right: 0.5em;
+			top: 0;
+			color: $grey-color;
```

Comment:
add `cursor: pointer;` to the closebtn class to make it more like a button (or don't because it's a prank anyway lol)

Reply:
That's a good idea, I think this is cruel enough with the `cursor: pointer` so I'll make the change quick